extension/pages/settings/preferences.css

```diff
@@ -152,6 +152,11 @@ body.dark #preferences a[href] {
 	color: green;
 }
 
+#preferences #addAllyFaction {
+	padding: 0 5px 0 5px;
+	width: fit-content;

```

Comment:
Firefox doesn't recognize `fit-content`, you'll also need `width: -moz-fit-content;`